core/templates/dev/head/error/disabled_exploration.html

```diff
@@ -4,9 +4,9 @@
 {% endblock navbar_breadcrumb %}
 
 <!-- TODO(mili) Translate the title. To avoid FOUC it must be done in the js -->
-{% block subtitle %}
-  Disabled Exploration
-{% endblock subtitle %}
+{% block maintitle %}
+  I18N_ERROR_DISABLED_EXPLORATION_PAGE_TITLE
```

Comment:
It seems odd to me that this does not follow the pattern of the other 'maintitle's -- this is an i18n code, whereas the others are strings. Is this an error, or am I missing something?